sdks/python/examples/dag/worker.py

```diff
@@ -26,26 +26,26 @@ def step1(input: EmptyModel, ctx: Context) -> StepOutput:
 
 
 @dag_workflow.task(execution_timeout=timedelta(seconds=5))
-def step2(input: EmptyModel, ctx: Context) -> StepOutput:
+async def step2(input: EmptyModel, ctx: Context) -> StepOutput:
     return StepOutput(random_number=random.randint(1, 100))
 
 
 @dag_workflow.task(parents=[step1, step2])
-def step3(input: EmptyModel, ctx: Context) -> RandomSum:
+async def step3(input: EmptyModel, ctx: Context) -> RandomSum:
```

Comment:
do we have any tested sync examples?

Reply:
yeah lots - this one is partially sync and partially async, just wanted to improve the coverage of `Context.task_output`